File: samples/WexflowTesting/ExecVb_src/Program1.vb
Imports System
Imports System.IO

Namespace ExecCsTest
    Module Program1
         Sub Main(args As String())
            Try
                File.Copy("c:\WexflowTesting\file1.txt", "c:\WexflowTesting\ExecVb_dest\file1.txt", True)
                Console.WriteLine("File copied!")
            Catch e As Exception
                Console.WriteLine(e)
            End Try
        End Sub
    End Module
End Namespace

File: samples/WexflowTesting/ExecCs_src/Program1.cs
using System;
using System.IO;

namespace ExecCsTest
{
    class Program1
    {
        static void Main(string[] args)
        {
            try
            {
                File.Copy(@"c:\WexflowTesting\file1.txt", @"c:\WexflowTesting\ExecCs_dest\file1.txt", true);
                Console.WriteLine("File copied!");
            }
            catch (Exception e)
            {
                Console.WriteLine(e);
            }
        }
    }
}
